app/assets/stylesheets/custom.css.scss

```diff
@@ -0,0 +1,111 @@
+@import "bootstrap-sprockets";
+@import "bootstrap";
+
+/* Variables */
+// COLORS
+// ======
+$light-gray: #777;
+$mega-light-gray: #eaeaea;
+$dark-gray: #555;
+$near-black: #222;
+$white: #fff;
+
+/* General Config */
+body {
+  padding-top: 60px;
+}
+
+section {
+  overflow: auto;
+}
+
+textarea {
+  resize: vertical;
+}
+
+.center {
+  text-align: center;
+
+  h1 {
+    margin-bottom: 10px;
+  }
+}
+
+/* Typography */
+h1, h2, h3, h4, h5, h6 {
+  line-height: 1;
+}
+
+h1 {
+  font-size: 3em;
+  letter-spacing: -2px;
+  margin-bottom: 30px;
+  text-align: center;
+}
+
+h2 {
+  font-size: 1.2em;
+  letter-spacing: -1px;
+  margin-bottom: 30px;
+  text-align: center;
+  font-weight: normal;
+  color: $light-gray;
+}
+
+p {
+  font-size: 1.1em;
+  line-height: 1.7em;
+}
+
+/* Header */
+#logo {
+  float: left;
+  margin-right: 10px;
+  font-size: 1.7em;
+  color: $white;
+  text-transform: uppercase;
+  letter-spacing: -1px;
+  padding-top: 9px;
+  font-weight: bold;
+}
+
+#logo:hover {
+  color: $white;
+  text-decoration: none;
+}
+
+.navbar {
+  background-color: #4BA448;
+  border-color: white;
+  color: #FFFFFF;
+}
+
+/* Footer */
+footer {
+  margin-top: 45px;
+  padding-top: 5px;
+  border-top: 1px solid $mega-light-gray;
+  color: $light-gray;
+
+  a {
+    color: $dark-gray;
+  }
+
+  a:hover {
+    color: $near-black;
+  }
+
+  small {
+    float: left;
+  }
+
+  ul {
+    float: right;
+    list-style: none;
+  }
+
+  ul li {
```

Comment:
Merge rule `ul li` with rule on line 102<br>Selector should have depth of applicability no greater than 2, but was 3